src/ParseServer.js

```diff
@@ -79,7 +81,19 @@ class ParseServer {
     Promise.all([dbInitPromise, hooksLoadPromise])
       .then(() => {
         if (serverStartComplete) {
-          serverStartComplete();
+          return serverStartComplete();
+        }
+        return Promise.resolve();
+      })
+      .then(() => {
+        if ((options.securityChecks || {}).enableLogOutput) {
```

Comment:
This should never be undefined, instead set a default value in option definitions.